cedar-policy-core/src/parser/cst_to_ast.rs

```diff
@@ -906,9 +1059,10 @@ fn build_ast_error_node_if_possible<Build: ExprBuilder>(error: ParseErrors) -> R
 /// Since ExprBuilder ErrorType can be Infallible or ParseErrors, if we get an error from building the node pass the ParseErrors along
 fn convert_expr_error_to_parse_error<Build: ExprBuilder>(
     error: ParseErrors,
+    #[cfg(feature = "tolerant-ast")] loc: Option<&Loc>,
```

Comment:
Is this a good idea? Especially when the argument is an option type?

Reply:
So the reason for this is to avoid an unused variable when not using the `tolerant-ast`. I don't see an issue with it but if you prefer I can just suppress the clippy error

Reply:
I'd rather suppress the warning then. Note that you can also suppress it conditionally with `#[cfg(not(feature = "tolerant-ast")), ...]` so it's easier to tell why it's there.

Reply:
Ah nice, I can suppress conditionally sure